spec/features/making_a_charge.rb

```diff
@@ -30,6 +29,20 @@
       let(:stripe_helper) { StripeMock.create_test_helper }
       before :each do
         StripeMock.start
+        within "#payment-form" do
+          fill_in :name, with: "Jeff"
+          fill_in :charge_email, with: "jeff@user.com"
+          fill_in :credit_card_number, with: '4242424242424242'
+          fill_in :exp_month, with: '10'
+          fill_in :exp_year, with: "2020"
+          fill_in :cvc, with: '123'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.